apps/gro/steps.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 
-var _ = require('lodash');
+const _ = require('lodash');
+const controllers = require('hof').controllers;
 
 module.exports = {
   '/': {
-    controller: require('../common/controllers/start'),
+    controller: controllers.start,
```

Comment:
I would rather you did `require('hof').controllers.start` here because we only use hof controller here and by calling the var `controllers` it could be misleading, i.e. someone might think it is all controllers